package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-confirm",
-  "version": "0.1.24",
+  "version": "0.2.0",
```

Comment:
My recommendation is to release this change as `0.2.0` to ensure users are aware that this is a breaking change.